ruby/QQ

```diff
@@ -0,0 +1,165 @@
+#Variables
+
+index = 0
+index_decrypt = 0
+proper_input = 0
+decoded_password = ""
+name = ""
+
+
+#Method definition
+
+def namescrambler(index_decrypt, name, decoded_password )
+
+    puts "what is your name"
+    name = gets.chomp
+    name_array = name.split
+
+    #split string into an array
+    reverse_name = name_array.reverse
+    #reverse the name's order
+    name_string = reverse_name.join(" ")
+    #convert array back to string
+
+      while index_decrypt < name_string.length
+      #isolate letter
+      letter_number_letter = name_string[index_decrypt]
+      #convert letter to the last letter
+
+      #lowercase vowels shifts
+      if letter_number_letter == "a"
```

Comment:
These statements are not dry

we should be collecting our letters in a single string.

```
vowels = "aeiou"
```

you can check to see if you letters are a part of those strings with `.include?()`